### ISSUE TITLE
ID 261815: [Programmatic Access - Simon - Search dashboards]: Alternate text or role="presentation" is not defined for the images present in the "Search dashboards" page.

### ISSUE THREAD

**Issue by eledobleefe** [grafana/grafana#66356]
### **User Experience:**  
Visually impaired users who depends on screen reader and navigates in scan mode will not be able to understand the information provided or purpose of the image.
_Note: User credentials should NOT be included in the bug._

### **Repro Steps:** 

1. Open #URL: [play.grafana.org](https://play.grafana.org/). Sign in with valid credentials. "Home" page appears. 
2. Navigate to the left navigation bar and select the  "Dashboard" button. 
3. New page appears, Select the "Playlists" tab item. 
4. Navigate the "Playlists" tab. 
5. Navigate to one of the created playlist and select "Start Playlist" button. 
6. Navigate the controls of "Start Playlist" pop-up Make necessary changes and select the "Start Lobby" button. 
7. Navigate the lobby page which appears. 
8. Navigate to the "Annotations" dashboard. 
9. Run AI tool. Verify the issue.

**Actual Result:**
Alternate text or `role="presentation"` is not defined for the images present in the "Search dashboards" page. 
Screen reader behavior: While navigating on the checkboxes screen reader (Narrator) announces as "Unlabeled graphic".

**Expected Result:**
Alternate text or role="presentation" should be defined for the images present in the "Search dashboards" page.

**Issue:**
Ensures <img> elements have alternate text or a role of none or presentation (image-alt - https://accessibilityinsights.io/info-examples/web/image-alt) 

**Target application:**  
Annotations - Grafana - https://play.grafana.org/d/000000010/annotations?orgId=1&search=open&query=&layout=grid 

**Element path:**  
`.css-15y6ntv:nth-child(1) > .css-1r8khxw > .css-1nsb508 > .css-mwhr24[loading="lazy"] `

**Snippet:** 
`<img loading="lazy" class="css-mwhr24" src="/api/dashboards/uid/vAFCy-onz/img/thumb/dark">`

How to fix:
Fix any of the following: 

- Element does not have an `alt` attribute 
- `aria-label` attribute does not exist or is empty 
- `aria-labelledby` attribute does not exist, references elements that do not exist or references elements that are empty 
- Element has no `title` attribute 
- Element's default semantics were not overridden with `role="none"` or `role="presentation"`

**Environment:**  
Microsoft Edge Version 105.0.1300.0
This accessibility issue was found using Accessibility Insights for Web 2.32.0 (axe-core 4.4.1), a tool that helps find and fix accessibility issues.  

### **Severity:**
`SEV2`

### **Area:**
`SearchCard component `(Dashboards)

**Comment by grafanabot** [comment]
@eledobleefe please add one or more appropriate labels. Here are some tips:

- if you are making an issue, TODO, or reminder for yourself or your team, please add one label that best describes the product or feature area. Please also add the issue to your project board. :rocket:

- if you are making an issue for any other reason (docs typo, you found a bug, etc), please add at least one label that best describes the product or feature that you are discussing (e.g. `area/alerting`, `datasource/loki`, `type/docs`, `type/bug`, etc). [Our issue triage](https://github.com/grafana/grafana/blob/main/ISSUE_TRIAGE.md#3-categorizing-an-issue) doc also provides additional guidance on labeling. :rocket:

 Thank you! :heart:

**Comment by mgifford** [comment]
This seems like a well fleshed out accessibility description to me. Accessibility Insights is a great tool for helping to provide reproducible and well documented results.
